functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobsSpec.groovy

```diff
@@ -6,28 +6,71 @@ import org.rundeck.util.annotations.APITest
 import org.rundeck.util.common.jobs.JobUtils
 import org.rundeck.util.container.BaseContainer
 
+/**
+ * Tests for the jobs API endpoint
+ */
 @APITest
 class JobsSpec extends BaseContainer {
 
+    static final def newProject = UUID.randomUUID().toString()
+    static List<String> createdJobIds
+
     def setupSpec() {
         startEnvironment()
         setupProject()
+
+        setupProject(newProject)
+
+        createdJobIds = createJobs()
     }
 
-    def "Listing RunDeck Jobs for project test"() {
+    def cleanupSpec() {
+        deleteProject(newProject)
+    }
+
+    def "Listing jobs for a project that has no jobs"() {
         when:
-            def response = doGet("/project/${PROJECT_NAME}/jobs")
+        def aProject = UUID.randomUUID().toString()
+        setupProject(aProject)
```

Comment:
It shouldn't be necessary to call setupProject here again

Reply:
That's another project being set up in addition to the one in setupSpec()